build.gradle

```diff
@@ -8,7 +8,6 @@ plugins {
 }
 
 repositories {
-    mavenLocal()
```

Comment:
I removed that to have more predictable builds. I guess no one here should use dependencies from there.

Reply:
I believe I included this because my typical test flow involves `publishToMavenLocal` and testing the new version of the plugin on some real example kotlin backend and android projects I have locally. In any case, that's something I can edit in ad hoc.

Reply:
I'm not sure I understood. `publishToMavenLocal` task doesn't rely on having `mavenLocal` repository. It only can be useful if you test locally published _Kotlinter_ on this project itself - does that take place? If it is, I'll revert that change - I didn't wan't to make anyone to change habbits.

I'm proposing to remove that as I couldn't see any use case here and basing on my experience and [Gradle documentation](https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:case-for-maven-local) having `mavenLocal` may lead to unpredictible builds (especially when repositories order is one of the factors in depenedency resolution process)

Reply:
My flow for testing plugins involves a separate repository, therefor I don't think you really need to have `mavenLocal` in the build file.